src/pages/settings/Report/RoomNamePage.js

```diff
@@ -100,14 +96,15 @@ function RoomNamePage(props) {
                     enabledWhenOffline
                 >
                     <View style={styles.mb4}>
-                        <RoomNameInput
-                            ref={(ref) => (roomNameInputRef.current = ref)}
+                        <InputWrapper
+                            InputComponent={RoomNameInput}
+                            ref={roomNameInputRef}
```

Comment:
This caused a prop warning on `RoomNameInput` `ref`. Please update `roomNameInputPropTypes` `forwardedRef` type to `refPropTypes`